module3-crud-and-data-models/r1.1-principles-of-setting-up-your-schema/README.md

```diff
@@ -0,0 +1,57 @@
+## Principles of setting up your schema
+
+Poorly designed databases can cause many problems, including wasting resources, making maintenance difficult, and hindering performance. That's why having a great database schema design is a crucial part of effective data management.
+By defining categories of data and relationships between those categories, database schema design makes data much easier to retrieve, consume, manipulate, and interpret.
+Without a clean, efficient, consistent database schema, you’ll struggle to make the best use of your enterprise data. For example, the same data might be duplicated in multiple locations—or even worse, might be inconsistent between these locations.
+
+### Type of database
+
+Most developers don't see the difference between relational database schema and MongoDB schema, but in reality, it is not the same
+
+<img src="https://i.imgur.com/3yobiPB.jpg" alt="meme" width="400"/>
```

Comment:
as i suggested to halit i would copy the image into the repository rather than linking to external images, as usually this is not very future proof (external image links can break for any number of reasons)